deps.edn

```diff
@@ -1,26 +1,61 @@
 {:paths ["src"]
- :deps {com.draines/postal                      {:mvn/version "2.0.5"},
-        com.flexiana/tiny-rbac                  {:mvn/version "0.1.1"},
-        com.taoensso/timbre                     {:mvn/version "5.1.2"},
-        crypto-password/crypto-password         {:mvn/version "0.3.0"},
-        funcool/cuerdas                         {:mvn/version "2.2.1"},
-        info.sunng/ring-jetty9-adapter          {:mvn/version "0.17.5"},
-        metosin/malli                           {:mvn/version "0.8.4"},
-        metosin/reitit                          {:mvn/version "0.5.18"},
-        migratus/migratus                       {:mvn/version "1.3.6"},
-        nilenso/honeysql-postgres               {:mvn/version "0.4.112"},
-        org.clojure/core.async                  {:mvn/version "1.5.648"}
-        org.clojure/data.json                   {:mvn/version "2.4.0"},
-        org.clojure/data.xml                    {:mvn/version "0.0.8"},
-        org.postgresql/postgresql               {:mvn/version "42.3.3"},
-        piotr-yuxuan/closeable-map              {:mvn/version "0.35.0"},
-        seancorfield/next.jdbc                  {:mvn/version "1.2.659"},
-        yogthos/config                          {:mvn/version "1.2.0"}}
+ :deps  {com.draines/postal              {:mvn/version "2.0.5"},
```

Comment:
why do we have both `deps.edn` and `project.clj` files in the project? shouldnt we stick to only using one so they dont get out of sync (at least from dependencies point of view)?

Reply:
@jehaby ping

Reply:
It was needed for a client project that used clojure cli. If I'm not mistaken it couldn't use the framework otherwise. 
I've create a card for that here: https://github.com/Flexiana/framework/projects/4#card-83465000

Reply:
yes, we should transform it in a clj-template ( https://github.com/seancorfield/clj-new)